src/templates/views/payment.hbs

```diff
@@ -41,35 +41,38 @@
 
             <h2>Shipping address</h2>
 
-            <input id="shipping-same" name="shipping-shipping-same" type="checkbox">
-            <label for="shipping-shipping-same">Same as billing address</label>
+            <details>
+                <summary>Enter shipping address (optional)</summary>
 
-            <label for="shipping-honorific-prefix">Title</label>
-            <input required autocomplete="honorific-prefix" id="shipping-honorific-prefix" name="shipping-honorific-prefix">
+                <input id="shipping-same" name="shipping-shipping-same" type="checkbox">
+                <label for="shipping-shipping-same">Same as billing address</label>
 
-            <label for="shipping-given-name">First name</label>
-            <input required autocomplete="given-name" id="shipping-given-name" name="shipping-given-name">
+                <label for="shipping-honorific-prefix">Title</label>
+                <input required autocomplete="honorific-prefix" id="shipping-honorific-prefix" name="shipping-honorific-prefix">
```

Comment:
These should not be required as this will prevent form submission in case of same address.